crates/forge/bin/cmd/test/mod.rs

```diff
@@ -814,6 +828,47 @@ fn persist_run_failures(config: &Config, outcome: &TestOutcome) {
     }
 }
 
+/// Generate test report in JUnit XML report format.
+fn junit_xml_report(results: &BTreeMap<String, SuiteResult>, verbosity: u8) -> Report {
+    let mut total_duration = Duration::default();
+    let mut junit_report = Report::new("Test run");
+    junit_report.set_timestamp(Utc::now());
+    for (suite_name, suite_result) in results {
+        let mut test_suite = TestSuite::new(suite_name);
+        total_duration += suite_result.duration;
+        test_suite.set_time(suite_result.duration);
+        test_suite.set_system_out(suite_result.summary());
+        for (test_name, test_result) in &suite_result.test_results {
+            let mut test_status = match test_result.status {
+                TestStatus::Success => TestCaseStatus::success(),
+                TestStatus::Failure => TestCaseStatus::non_success(NonSuccessKind::Failure),
+                TestStatus::Skipped => TestCaseStatus::skipped(),
+            };
+            test_status.set_message(test_result.reason.clone().unwrap_or_default());
```

Comment:
let's only set if Some

Reply:
:+1: changed in https://github.com/foundry-rs/foundry/pull/8852/commits/c8d9c52c875e2d7931a8b523b394895d6d7c6806